inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/config/RemoteConfigManager.java

```diff
@@ -143,8 +143,12 @@ public void reload() {
         //
         this.ipListParser.setCommonProperties(ConfigManager.getInstance().getCommonProperties());
         List<String> managerIpList = this.ipListParser.getIpList();
+        if (managerIpList == null || managerIpList.size() == 0) {
+            return;
```

Comment:
directly return， but does not print log "end to reload config".

Reply:
add error log